### PR TITLE
CLOUDY-679 moving rancher instances to private subnet

### PR DESCRIPTION
**Pull request by eddiechristian**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -166,6 +166,12 @@ variable "rancher2_worker_subnet_ids" {
   description = "List of subnet ids for Rancher worker nodes"
 }
 
+variable "lb_subnet_ids" {
```
Comment (patrickpierson) [inline variables.tf:169]: is this something that we add to local.tf.vars?

File: rke.tf [hunk 2]
```diff
@@ -4,21 +4,29 @@
 resource "rke_cluster" "rancher_server" {
   depends_on = [null_resource.wait_for_docker]
 
+  bastion_host {
```
Comment (patrickpierson) [inline rke.tf:7]: is this just an instance or an asg?

File: variables.tf (hunk 1 above)
Comment (eddiechristian) [reply variables.tf:169]: no it actually comes from vpc_module here``
`module "rancher_server" {
  source = "git@github.com:IronNetCybersecurity/terraform-rancher-server.git?ref=CLOUDY-679"
  name   = local.name
  # rancher_current_password   = var.rancher_current_password
  rancher_password           = var.rancher_password
  use_default_vpc            = false
  vpc_id                     = module.rancher_vpc.vpc_id
  aws_region                 = var.aws_region
  aws_profile                = null
  aws_elb_subnet_ids         = module.rancher_vpc.public_subnets
  domain                     = var.rancher_domain
  r53_domain                 = local.rancher_r53_domain
  rancher2_master_subnet_ids = module.rancher_vpc.private_subnets
  rancher2_worker_subnet_ids = module.rancher_vpc.private_subnets
  lb_subnet_ids              = module.rancher_vpc.public_subnets
  rancher2_custom_tags       = local.common_tags
  rancher_ssh_ingress_cidr   = local.whitelist_cidrs
  bastion_dns                = aws_route53_record.bastion_dns_lb.fqdn
  master_node_count          = 3
  worker_node_count          = 3

  providers = {
    aws     = "aws"
    aws.r53 = "aws.r53"
  }
}`

Comment (patrickpierson) [reply variables.tf:169]: got it.

File: rke.tf (hunk 2 above)
Comment (eddiechristian) [reply rke.tf:7]: it just points to the instance we created with bastion module

Comment (patrickpierson) [reply rke.tf:7]: ok

**Review by patrickpierson — APPROVED** [review]

**Pull request merged**